includes/sanitizers/class-amp-iframe-sanitizer.php

```diff
@@ -135,6 +136,7 @@ private function filter_attributes( $attributes ) {
 				case 'sandbox':
 				case 'class':
 				case 'sizes':
+				case 'id':
```

Comment:
Instead of just allowing `id` we should probably obtain all attributes that are allowed on `amp-iframe`, including `data-*` ones. For non-`data-*` attributes, the list of attributes can be obtained via `AMP_Allowed_Tags_Generated::get_allowed_tag( 'amp-iframe' )`. The list of globally-allowed attributes can be obtained via `AMP_Allowed_Tags_Generated::get_allowed_attributes()`...

On second thought, we it would be better to just copy _all_ of the attributes over from the `iframe` because then the whitelist sanitizer can then report when invalid attributes are present, as opposed to silently stripping them out.